app/src/components/preloaders/spinningPreloader/spinningPreloader.scss

```diff
@@ -19,6 +19,7 @@
   position: relative;
   width: 100%;
   height: 100%;
+  background-color: $COLOR--white-two;
```

Comment:
I think will be better to leave this component without background as it can be used in different places with different backgrounds and may inherit it from parents.